net-analyzer/gsad/gsad-24.1.0.ebuild

```diff
@@ -60,6 +60,16 @@ src_prepare() {
 		   done
 		fi
 	fi
+
+	# Avoid the use of brotli when not required by the use flag #942193
+	# Remove brotli automagic dependencies check
+	if ! use brotli; then
```

Comment:
I'm afraid this looks quite brittle and is vulnerable to silently not doing anything in future. This would be better with a patch so you at least get some notification (when it stops applying) if the surrounding code changed.

But also, `-DCMAKE_DISABLE_FIND_PACKAGE_BROTLI=$(usex !brotli)` may work in `src_configure`.

Reply:
Thanks for the pointer to `CMAKE_DISABLE_FIND_PACKAGE`. I did not know that such a thing existed.